src/client/pages/Path/Path.jsx

```diff
@@ -93,8 +93,8 @@ LessonCard.propTypes = {
   type: PropTypes.string.isRequired,
 };*/
 CourseCard.propTypes = {
-  pathId: PropTypes.objectOf(PropTypes.shape).isRequired,
-  courseId: PropTypes.objectOf(PropTypes.shape).isRequired,
+  pathId: PropTypes.string.isRequired,
+  courseId: PropTypes.string.isRequired,
   curriculum: PropTypes.objectOf(PropTypes.shape).isRequired,
```

Comment:
`curriculum: PropTypes.objectOf(PropTypes.shape).isRequire` it should be the other say around
`curriculum: PropTypes.shapeOf(PropTypes.object).isRequired`